src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectLoader.cs

```diff
@@ -362,7 +362,8 @@ private async Task<bool> ReloadProjectAsync(ProjectToLoad projectToLoad, ToastEr
                 _projectSystemHostInfo);
 
             var loadedProject = new LoadedProject(projectSystemProject, projectFactory, _fileChangeWatcher, _targetFrameworkManager);
-            loadedProject.NeedsReload += (_, _) => _projectsToReload.AddWork(projectToLoad with { ReportTelemetry = false });
+            loadedProject.NeedsReload += (_, _) =>
+                _projectsToReload.AddWork(projectToLoad with { ReportTelemetry = false });
```

Comment:
Was this accidentally left from some editing you were doing? It's not bad, but given the line before is longer it's a bit strange...